assets/js/localization.js

```diff
@@ -383,7 +412,7 @@ function localizeInterface() {
     });
 
     function direction(lang) {
-        return rtlLanguages.indexOf(lang) === -1 ? 'ltr' : 'rtl';
+        return rtlLanguages.indexOf(lang) !== -1 ? 'rtl' : 'ltr';
```

Comment:
Ohh damn.. while removing the function, I forgot the original form :P. I'll edit this now.